app/controllers/openstax/api/v1/api_controller.rb

```diff
@@ -24,6 +30,7 @@ class ApiController < ActionController::Base
         skip_before_filter :verify_authenticity_token, unless: :session_user?
         skip_before_filter :verify_authenticity_token, only: :options
         before_filter :set_cors_preflight_headers, only: :options
+        before_filter :set_cors_headers
         after_filter :set_cors_headers
 
```

Comment:
I don't know.  I assumed an `after_filter` was used on purpose because something mid-request was putting in defaults or something.  I have a recollection of that specifically for dates, but honestly I don't know.  Would kind of hate to remove them without knowing the impact.